src/Sulu/Bundle/CategoryBundle/Api/Category.php

```diff
@@ -460,20 +461,23 @@ private function getUserFullName($user)
     /**
      * Takes an array of CollectionMeta and returns a single meta for a given id.
      *
-     * @param $meta
-     * @param $id
+     * @param CollectionMeta $meta
+     * @param int $id
      *
-     * @return CollectionMeta
+     * @return null|CollectionMeta
      */
     private function getSingleMetaById($meta, $id)
     {
         if (null !== $id) {
+            /** @var CollectionMeta $singleMeta */
```

Comment:
this should not be necessary if we use `CategoryMetaInterface[]` for the $meta param